src/Microsoft.DotNet.Interactive.VSCode/KernelExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Interactive.Commands;
+using Microsoft.DotNet.Interactive.CSharp;
+using Microsoft.DotNet.Interactive.FSharp;
+
+namespace Microsoft.DotNet.Interactive.VSCode
+{
+    public static class KernelExtensions
+    {
+        public static CSharpKernel UseVSCodeHelpers(this CSharpKernel kernel)
+        {
+            var command = new SubmitCode($@"
+#r ""{typeof(VSCodeInteractiveHost).Assembly.Location.Replace("\\", "/")}""
+{typeof(IInteractiveHost).FullName} host = new {typeof(VSCodeInteractiveHost).FullName}();
```

Comment:
I'm not sure we should have this `host` variable in the long term.

That said, `Host` would be a more conventional name.

Reply:
Renamed to `InteractiveHost` and also made it work with PowerShell.